tests/test_libraries.py

```diff
@@ -2078,6 +2078,7 @@ def test_xarray(pyi_builder):
             dims=("x", "y"),
             coords={"x": [10, 20]},
         )
+        data_zarr = data.to_zarr()
```

Comment:
Ah, that's not what I had I mind. I meant to create a copy of `test_xarray` (named `test_xarray_to_zarr`), which has the added conversion to `zarr`. And this new test requires both `@importorskip('xarray')` and `@importorskip('zarr')`.